oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/JdbcOAuth2AuthorizationService.java

```diff
@@ -551,6 +610,25 @@ private String writeMap(Map<String, Object> data) {
 
 	}
 
+	public static class DefaultColumnTypeExtractor implements Function<JdbcOperations, Integer> {
```

Comment:
Let's change this to a `private static` utility method, e.g. `private static int getColumnDataType(JdbcOperations jdbcOperations, String columnName)`